syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -443,6 +447,70 @@ def matmul(self, *args, **kwargs):
     __matmul__ = matmul
     mm = matmul
 
+    # Approximations:
+    def exp(self, iterations=8):
```

Comment:
Same with both below functions

Reply:
These should also have citations, since the inspiration comes from CrypTen. I'd say it's not substantial enough to require that the license be copied, so a simple online comment + link should do

Reply:
Indeed, but I think this is a good temporary place until polynomial is ready to receive them and we have a clear strategy to integrate Poly & FixedP.
Also, I've added references to the repo where I analyse, test and adapt those approximations suggested by crypten.

Reply:
Thank you for doing that @LaRiffle and @jvmancuso